### DISCUSSION TITLE
Game is too intense on CPU

### DISCUSSION THREAD

**Discussion by mmikeyy** [voxeltycoon/issues#1042] [Troubleshooting]
When I play, all fans start spinning at max speed, even on a paused game. I've now played over 120 hours in the last 2 weeks (this game is so addictive!) and I'm starting having problems. It seems to have to do with the number of train and truck routes. The processor temperature reaches 100C and stays at that level or slightly below. Suddenly, poof... the computer shuts off. It can happen after half an hour, perhaps even an hour, but it seems to reliably happen now (last 2-3 days). It never did before. 

I've played over 1000 hours of Factorio without problems. Hundreds of hours of Satisfactory without problems. 

Could there be something wrong with the Unity engine? Does it have to do with how the game is programmed? Are all vehicles animated even when off screen? Are paths recalculated for all trains each game cycle, even for trains moving between the same two signals from one game cycle to the next? Is the CPU doing things the GPU should be doing instead? Whatever the reason, this is a big problem and I fear that it might reach a point where I'll just need to set this game aside. I hope not, because it is truly a fantastic game!

BTW I went in the Windows task manager and put the game process in 'economy' mode. Core temperature went way down and the problem disappeared. But panning became jerky, except on a paused game. Not fun. I tried with no shadows, no anti-aliasing, and the game is still cooking my processor. Sigh... As I'm typing this, the game is paused, music silent as the cursor is not even on it, and the processor stays between 95C and 100C. What the hell is it doing???

**Comment by Higuys153** [comment]
what cooling setup do you have and what are your pc specs
